nucypher/policy/policies.py

```diff
@@ -160,6 +165,29 @@ def payload(self):
         return bytes(self.publish_transaction) + partial_payload
 
 
+class PolicyPayloadMutex(DeferredList):
+
+    def __init__(self, deferredList, percent_to_complete_before_release=5, *args, **kwargs):
+        self.percent_to_complete_before_release = percent_to_complete_before_release
+        self._policy_locking_queue = Queue()
+
+        super().__init__(deferredList, *args, **kwargs)
+        self._block_until_this_many_are_complete = int(len(deferredList) / self.percent_to_complete_before_release)
```

Comment:
Seems like the name `percent_to_complete_before_release` is misleading: I expected it to act as 
```
int(len(deferredList) * self.percent_to_complete_before_release / 100)
```

Reply:
Ugh, you're right.  That's a blunder.

Reply:
@jMyles - has this one been resolved?